common/fs/fs.go

```diff
@@ -0,0 +1,52 @@
+package fs
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"syscall"
+)
+
+// T is the filesystem interface type.
+type T interface {
```

Comment:
In the `sort` package (and other places?) this is called `type Interface interface {...}`. Not sure if I like that, but it's a bit more descriptive than `T`.